### ISSUE TITLE
how to localize / translate WallacePOS.

### ISSUE THREAD

**Issue by humansoft** [micwallace/wallacepos#138]
Hi guys
I like WalacePOS.
I prefer offline support and mobile response.
I want to translate it into another language and use it as another currency, but I do not know what to do.
I'm not sure I need to translate all the source files.
I would appreciate it if you could guide me through it.
Thank you.

**Comment by humansoft** [comment]
Hi,
I do not know because I am a beginner, but to translate this program it seems that I need to modify all the source programs.
I hope it will be upgraded to translate only the replacement of the language file without touching the source file in the future.
Thank you.

**Comment by GAbbona** [comment]
Yes, please. It would be wonderful to be able to make translation easy.
